beetsplug/hook.py

```diff
@@ -58,7 +58,7 @@ def convert_field(self, value, conversion):
         converted = super(CodingFormatter, self).convert_field(value,
                                                                conversion)
         try:
-            converted = converted.decode(self._coding)
+            converted = converted.encode('utf-8').decode(self._coding)
```

Comment:
I don't quite see what's going on here—shouldn't `format_string` and `converted` be bytestrings here?